File: Tema1/ErrorAbsoluto.py
class ErrorAbsoluto:
    @staticmethod
    def calcular(valor_real, valor_medido):
        return abs(valor_real - valor_medido)


if __name__ == "__main__":
    ea = ErrorAbsoluto.calcular(2.50, 2.45)
    print(f"Error absoluto: {ea}")

File: tema1/ErrorAbsoluto.java
public class ErrorAbsoluto {
    public static double calcular(double valorReal, double valorMedido) {
        return Math.abs(valorReal - valorMedido);
    }

    public static void main(String[] args) {
        double error = ErrorAbsoluto.calcular(2.50, 2.45);
        System.out.printf("Error absoluto: %.3f%n", error);
    }
}
